<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1006</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="mainview">
    <property name="geometry">
     <rect>
      <x>353</x>
      <y>2</y>
      <width>651</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="other">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>351</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGraphicsView" name="bg_color">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>207</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="angle_step">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>477</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Количество </string>
     </property>
    </widget>
    <widget class="QPushButton" name="draw_once">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>387</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отобразить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="draw_centr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>507</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Концентр</string>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_6">
     <property name="geometry">
      <rect>
       <x>144</x>
       <y>357</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>A</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clean_all">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>567</y>
       <width>351</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Очистить холст</string>
     </property>
    </widget>
    <widget class="QLabel" name="diametr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>447</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Шаг </string>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_7">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>358</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>B</string>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>237</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>167</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Цвет линии</string>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>322</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Радиус </string>
     </property>
    </widget>
    <widget class="QLabel" name="end_point">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>257</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Центр</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="ln_color">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>207</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_bground">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>167</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Цвет фона</string>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>550</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>295</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Для окружности:</string>
     </property>
    </widget>
    <widget class="QLabel" name="begin_point_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Для эллипса:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="lib">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Библиотечный</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="brez">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Брезенхем</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="canon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Каноническое уравнение </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="param">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Параметрическое уравнение </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="middle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Средней точки</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dia">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>447</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="count">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>477</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="rad">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>320</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="a">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>356</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="b">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>356</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="centr_x">
     <property name="geometry">
      <rect>
       <x>189</x>
       <y>256</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="centr_y">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>256</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="color_line_like_bg_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>167</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Цвет линии как у фона</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="what">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="circle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Окружность </string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ellips">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>25</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Эллипс </string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
